src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx

```diff
@@ -286,6 +286,7 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
 
     const actions = this.state.actions;
     const currentAction = this.state.currentAction;
+    const fnDescription = () => (<span>Short description of the proposal.<ul><li>What are you proposing to do?</li><li>Why is it important?</li><li>How much will it cost the DAO?</li><li>When do you plan to deliver the work?</li></ul></span>);
```

Comment:
Is there a place we can hold constant text like this?
The code can look cleaner if it'll will be fetched from a constants file for example.
Also, I think we use this text in more places so moving this to a separate file can reduce code duplication.

Reply:
> Is there a place we can hold constant text like this?

I would like to do that too, this is often-duplicated code.  I'd say in proposalUtils.  But not in this PR.

Reply:
agree , lets move it to a some lable.json somehow.

Reply:
in a separate PR.  not needed in 1.0

Reply:
in any case, I think putting this in a json file may be overkill, over-engineering unless we can show a need for it.

Reply:
note that fnDescription  needs to be a component...

Reply:
moving all texts to specific file  is much cleaner code and well engineered.
will handle that on 2.0.